src/atom.xml

```diff
@@ -13,17 +13,31 @@ title: Atom Feed
         <email>{{ site.email }}</email>
     </author>
     <rights>Copyright (c) {{ site.sitename }}</rights>
-    {% for post in site.posts limit:15 %}
+    {%- for post in site.posts limit:15 -%}
```

Comment:
Why the hyphens?

Reply:
suppresses the newlines https://shopify.github.io/liquid/basics/whitespace/

Reply:
Oooh